cmd/kubeadm/app/cmd/upgrade/diff.go

```diff
@@ -86,7 +87,8 @@ func runDiff(flags *diffFlags, args []string) error {
 	if flags.cfgPath != "" {
 		cfg, err = configutil.LoadInitConfigurationFromFile(flags.cfgPath)
 	} else {
-		client, err := kubeconfigutil.ClientSetFromFile(flags.kubeConfigPath)
+		var client *client.Clientset
+		client, err = kubeconfigutil.ClientSetFromFile(flags.kubeConfigPath)
```

Comment:
For reference, this is _not_ the error I am talking about (this is just where the new `err` is/was declared). The error not previously handled is the one on line 95 https://github.com/kubernetes/kubernetes/pull/80648/files#diff-a6dcbd409dc17bd39fbe918f4bf5ee5fR95